packages/core/types/src/types/core/attributes/base.ts

```diff
@@ -1,3 +1,15 @@
+// TODO: Import it from @strapi/database but is not exported
```

Comment:
Does it need to be `@experimental`? Can `@deprecated` also work because it gives some visual cues in the IDE?! 🤔

Reply:
https://github.com/microsoft/TypeScript/issues/56808

Reply:
As per
>For now, a workaround is to use @deprecated, then clearly denote that the feature will be un-deprecated in a following release, **but some people might misinterpret that or not read the whole message**:

I would rather keep experimental to avoid the confusion about deprecation

Reply:
Although, as a compromise, we could keep both for the moment (basically `@experimental` + `@deprecated <reason>`)